readme.md

```diff
@@ -1,7 +1,9 @@
 keycloak identiy management server with mysql mariadb on docker 
 
```

Comment:
Add few high-level information on Keycloak.
Or create a wiki to explain the minimal setup information

Reply:
updated